inductiva/fluids/scenarios/wind_tunnel/post_processing.py

```diff
@@ -40,47 +39,99 @@ def __init__(self,
 
         self.sim_output_dir = sim_output_path
-        self.object_path = self.get_object
-        self.object_data = self.get_object_data(simulator, time_step)
-        self.pressure_field = self.get_pressure_field(simulator)
-
-    @singledispatchmethod
-    def get_object_data(self, simulator: Simulator, time_step: int): # pylint: disable=unused-argument
-        return ValueError(
-            f"Simulator not supported for `{self.__class__.__name__}` scenario."
-        )
-
-    @singledispatchmethod
-    def get_pressure_field(self, simulator: Simulator): # pylint: disable=unused-argument
-        return ValueError(
-            f"Simulator not supported for `{self.__class__.__name__}` scenario."
-        )
-
-
-@WindTunnelSimulationOutput.get_object_data.register
-def _(self, simulator: OpenFOAM, time_step: int): # pylint: disable=unused-argument
-    """Get data over object for OpenFOAM."""
-
-    reading_file = os.path.join(self.sim_output_dir, "foam.foam")
+        self.object_data = self.get_object_data(time_step)
 
-    # Create reading file
-    with open(reading_file, "w", encoding="utf-8") as file:
-        file.close()
+    def get_object_data(self, time_step: int): # pylint: disable=unused-argument
+        """Get aerodynamics data over an object set inside the WindTunnel
+        
+        Current Support - OpenFOAM
 
-    # Initialize reader and define reading time-step
-    reader = pv.POpenFOAMReader(reading_file)
-    reader.set_active_time_value(time_step)
+        Args:
+            time_step: Time step for which we want to visualize
+                aerodynamics data.
+        """
 
-    mesh = reader.read()
-    object_data = mesh["boundary"]["object"]
+        reading_file = os.path.join(self.sim_output_dir, "foam.foam")
 
-    return object_data
+        # Create reading file
+        with open(reading_file, "w", encoding="utf-8") as file:
+            file.close()
 
+        # Initialize reader and define reading time-step
+        reader = pv.POpenFOAMReader(reading_file)
+        reader.set_active_time_value(time_step)
 
-@WindTunnelSimulationOutput.get_pressure_field.register
-def _(self, simulator: OpenFOAM): # pylint: disable=unused-argument
-    """Returns pressure field over mesh points at a certain time_step."""
+        mesh = reader.read()
+        object_data = mesh["boundary"]["object"]
 
-    pressure_field = Object(self.object_data, "p")
+        return object_data
 
-    return pressure_field
+    def get_pressure_field(self): # pylint: disable=unused-argument
+        """Get pressure field over mesh points for a certain time_step.
+        
+        Returns:
+            A MeshData object that allow to manipulate data over a mesh
+            and to render it.
+        """
+        pressure_field = MeshData(self.object_data, "p")
+
+        return pressure_field
+
+    def get_streamlines(self,
+                        time_step: int,
+                        physical_property: str = "pressure"):
```

Comment:
If there are only two options for physical property, maybe it is better to create a list of options here, and let the user insert one.